### DISCUSSION TITLE
Terminal tab title left with last filename after exiting

### DISCUSSION THREAD

**Discussion by gwww** [LunarVim/LunarVim#3550] [General]
Whenever I edit a new file the title in the tab of my terminal changes to the filename. OK, I guess (although I would prefer it left my terminal tab text alone, if that's possible). My problem is that when I exit lunarvim it leaves the text of the last file there instead of changing back to the default program (zsh in my case). I'm using `wezterm` if that makes a difference. I'm running lunarvim 1.2 latest as of about 2 days ago.

**Comment by LostInTheLogs** [comment]
can you send a screenshot of what that looks like?

**Comment by gwww** [comment]
The green bit is the terminal title that is set on Wezterm tab 2. The image is while in LV but the title does not change after I exit LV. I be just as happy if LV didn't change my terminal title at all, although I get why some people might like it.

<img width="333" alt="Screen Shot 2022-11-30 at 12 22 09" src="https://user-images.githubusercontent.com/3060128/204865556-4297b87c-fe63-4678-97ee-573312ed33ab.png">

**Reply by LostInTheLogs** [reply]
you can try `vim.opt.title = false`

**Reply by gwww** [reply]
Thanks! That works. In all my years of vim/nvim I had never come across the setting.
